File: solution/ServerExam/Entity/SubjectEntity.cs
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace ServerExam.Entity
{
    public class SubjectEntity
    {
        [Key]
        [JsonPropertyName("ID")]
        public int ID { get; set; }

        /// <summary>
        /// 科目名称
        /// </summary>
        [JsonPropertyName("SubjectName")]
        public string SubjectName { get; set; }

        /// <summary>
        /// 科目代码
        /// </summary>
        [JsonPropertyName("SubjectCode")]
        public string SubjectCode { get; set; }

        /// <summary>
        /// 科目状态 1正常 2禁用
        /// </summary>
        [JsonPropertyName("SubjectState")]
        public int SubjectState { get; set; }

        /// <summary>
        /// 创建时间
        /// </summary>
        [JsonPropertyName("CreateTime")]
        public int CreateTime { get; set; }

        /// <summary>
        /// 更新时间
        /// </summary>
        [JsonPropertyName("UpdateTime")]
        public int UpdateTime { get; set; }

        public SubjectEntity()
        {
            this.ID = 0;
            this.SubjectName = "";
            this.SubjectCode = "";
            this.SubjectState = 0;
            this.CreateTime = 0;
            this.UpdateTime = 0;
        }
    }
}

File: solution/Entity/SubjectEntity.py
# -*- coding:utf-8 -*-
from Entity.BaseEntity import *


# 科目
class SubjectEntity(BaseEntity, BaseORM):
    __tablename__ = 'Subject'

    SubjectName: str = Column(String(128), comment='科目名称', default='none')
    SubjectCode: str = Column(String(128), index=True, comment='科目代码', default='none')
    SubjectState: int = Column(INTEGER, comment='科目状态 1正常 2禁用', default=0)
    UpdateTime: int = Column(INTEGER, comment='更新时间', default=int(time()))

    def __init__(self):
        super().__init__()